test/ModuleInterface/project-name.swift

```diff
@@ -0,0 +1,23 @@
+// RUN: %empty-directory(%t)
+
+// RUN: %target-swift-frontend -emit-module %s -I %t \
+// RUN:   -module-name Library -project-name ProjectName \
```

Comment:
This `-project-name` naming is confusing given that we have `-package-name` which sounds like a similar thing.  Given that `-project-name` was added very recently, could we rename it to something less confusing?

Reply:
That's a fair point, but I'd like to keep that the way it is for this PR. We can discuss changing that separately with @nkcsgexi.